js/page-specific/trails.js

```diff
@@ -1,907 +1,982 @@
+                }
 
-        $('.page-info').html(" | Page: " + pageInfo.currentPage + " of " + totalPages);
-        $('.records-info').html(recordMessage);
-    },
-    updatePageInformation = function (objectIds, page) {
-        pageInfo = {
-            objectIds: objectIds,
-            totalRecords: objectIds.length,
-            totalPages: function () {
-                var _this = this;
-                return Math.ceil(objectIds.length / _this.recordsPerPage);
-            },
-            currentRange: function () {
-                var _this = this,
-                    low = _this.currentPage === 1 ? 1 : (_this.currentPage - 1) * _this.recordsPerPage + 1,
-                    high = _this.recordsPerPage * _this.currentPage > _this.totalRecords ? _this.totalRecords : _this.recordsPerPage * _this.currentPage;
-                return low + " to " + high;
-            },
-            currentPage: page || 0,
-            recordsPerPage: 5
-        };
+                $('.page-info').html(" | Page: " + pageInfo.currentPage + " of " + totalPages);
+                $('.records-info').html(recordMessage);
+            },
+            updatePageInformation = function (objectIds, page) {
+                pageInfo = {
+                    objectIds: objectIds,
+                    totalRecords: objectIds.length,
+                    totalPages: function () {
+                        var _this = this;
+                        return Math.ceil(objectIds.length / _this.recordsPerPage);
+                    },
+                    currentRange: function () {
+                        var _this = this,
+                            low = _this.currentPage === 1 ? 1 : (_this.currentPage - 1) * _this.recordsPerPage + 1,
+                            high = _this.recordsPerPage * _this.currentPage > _this.totalRecords ? _this.totalRecords : _this.recordsPerPage * _this.currentPage;
+                        return low + " to " + high;
+                    },
+                    currentPage: page || 0,
+                    recordsPerPage: 5
+                };
+
+                if (pageInfo.currentPage > pageInfo.totalPages()) {
+                    queryRecordsByPage(pageInfo.currentPage - 1);
+                }
+            };
 
-        if (pageInfo.currentPage > pageInfo.totalPages()) {
+        /*Events*/
+        $(document).on("click", '.js-prev-btn', function (e) {
+            e.preventDefault();
             queryRecordsByPage(pageInfo.currentPage - 1);
-        }
-    };
 
-    /*Events*/
-    $(document).on("click", '.js-prev-btn', function (e) {
-        e.preventDefault();
-        queryRecordsByPage(pageInfo.currentPage - 1);
+            window.location.hash = pageInfo.currentPage === 1 ? "page-1" : "page-" + (pageInfo.currentPage);
+        });
 
-        window.location.hash = pageInfo.currentPage === 1 ? "page-1" : "page-" + (pageInfo.currentPage);
-    });
+        $(document).on("click", '.js-next-btn', function (e
```

Comment:
I think $this is a little misleading. In these cases, I usually call them $eventTarget or something similar.

Reply:
Also, did you mean to use evt.target or evt.currentTarget? This may be fine as-is, but it's worth thinking about.